### DISCUSSION TITLE
Question: It says I don't have any animations for my characters when I do

### DISCUSSION THREAD

**Discussion by ElbryanWyndon** [FunkinCrew/Funkin#1550] [Q&A]
I followed the tutorial by BrightFyre but it still tells me this
![image](https://user-images.githubusercontent.com/83390237/118012485-f9145600-b305-11eb-8fee-c7526d80befd.png)

**Comment by ElbryanWyndon** [comment]
also i dont know why it shows me the preview character thing for senpai, can i fix that in any way either?

**Comment by ElbryanWyndon** [comment]
![image](https://user-images.githubusercontent.com/83390237/118035683-6503b800-b320-11eb-8150-cbcba3b44f8a.png)
IN FREEPLAY IT EVEN USES THE RIGHT ICONS LIKE WHAT

**Comment by ElbryanWyndon** [comment]
if it matters i got an error when i tried to debug it in saying it couldnt find Voices.mp3 or Inst.mp3

**Comment by ghost** [comment]
You need to convert your .ogg files to mp3

**Comment by ElbryanWyndon** [comment]
what does that have to do with the animations

**Comment by ElbryanWyndon** [comment]
also i'll try that, btw im using renamed bopeebo, fresh, and dad battle until the music is done

**Comment by ElbryanWyndon** [comment]
NEW THING 
![image](https://user-images.githubusercontent.com/83390237/118042562-1870aa80-b329-11eb-8a97-45bfb24b4ee7.png) But i dont know what to make of it, burning_voices and burning_inst are all there

**Comment by ElbryanWyndon** [comment]
btw i used debug to get it

**Comment by ElbryanWyndon** [comment]
OH also freeplay mode deosnt give me any errors, i mean it doesnt work but it doesnt give me any errors

**Comment by ElbryanWyndon** [comment]
btw it says this even though i did what it asked 
![image](https://user-images.githubusercontent.com/83390237/118044914-1bb96580-b32c-11eb-9409-5cd2236a879c.png) heres proof too 
![image](https://user-images.githubusercontent.com/83390237/118044965-2aa01800-b32c-11eb-994f-57f9ba5ed505.png)
![image](https://user-images.githubusercontent.com/83390237/118045015-3c81bb00-b32c-11eb-96ae-14859ea45b0c.png)

**Comment by ElbryanWyndon** [comment]
am i doing it wrong

**Comment by NotHyper-474** [comment]
> ![image](https://user-images.githubusercontent.com/83390237/118042562-1870aa80-b329-11eb-8a97-45bfb24b4ee7.png) [...] burning_voices and burning_inst are all there

You said burnings_voices and burning_inst, so does that mean the Voices and Inst files are named that way? Because, they shouldn't, since the game loads the songs like this: "assets/songs/(song name all lowercased)/(Inst or Voices).mp3"

**Comment by ghost** [comment]
You did it shared/music. Put it in songs
